static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Protein-protein interaction graph
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import React from "react";
+
+import { drawProteinInteractionGraph } from "./chart";
+import { bioDcid, MultiLevelInteractionGraphData } from "./types";
+
+interface InteractionGraphProps {
+  centerProteinDcid: bioDcid;
+}
+
+interface InteractionGraphState {
+  depth: number;
+  graphData: MultiLevelInteractionGraphData;
+  numInteractions: number;
+  scoreThreshold: number;
+}
+
+const CHART_ID = "protein-interaction-graph";
+
+const DEFAULTS = {
+  DEPTH: 2,
+  MAX_INTERACTIONS: 4,
+  MISSING_SCORE_FILLER: -1,
+  SCORE_THRESHOLD: 0.4,
+};
+
+export class ProteinProteinInteractionGraph extends React.Component<
+  InteractionGraphProps,
+  InteractionGraphState
+> {
+  constructor(props: InteractionGraphProps) {
+    super(props);
+    this.state = {
+      depth: DEFAULTS.DEPTH,
+      graphData: null,
+      scoreThreshold: DEFAULTS.SCORE_THRESHOLD,
+      numInteractions: DEFAULTS.MAX_INTERACTIONS,
+    };
+  }
+
+  /**
+   * Perform BFS and (re)draw graph
```

Comment:
I don't think this performs BFS anymore?

Reply:
Oops that's right! thanks for catching!